templates/data-sources/git_repository.md.tmpl

```diff
@@ -0,0 +1,35 @@
+---
+Title: "Git Repository Data Source"
+Description: |-
+    Creating the Git Repository Data Source.
+---
+
+# Git Repository
+
+The `tanzu-mission-control_git_repository` Data Source allows you to get git repository to a particular scope through Tanzu Mission Control.
```

Comment:
Get this reviewed with the PDS team and add some more description under this block. At least 3-4 lines should be there. Similarly, in resources template file too.
cc: @ishangupta-ds

Reply:
Updated

Reply:
I don't see the updated description under this heading